packages/net/_test.pony

```diff
@@ -314,6 +314,9 @@ class _TestTCPWritevNotifyServer is TCPConnectionNotify
     end
     true
 
+  fun ref connect_failed(conn: TCPConnection ref) =>
+    None
```

Comment:
@kcrimson I think this issue is still outstanding - we should be calling `_h.fail` here to fail the test, instead of ignoring the connection failure.